components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/definitions/OAS3Parser.java

```diff
@@ -690,6 +690,37 @@ private void populatePathParameters(Operation operation, String pathName) {
         }
     }
 
+    /**
+     * Include authorizationUrl and scopes, mandatory elements to the OpenAPI definition if not present under
+     * securitySchemes in provided definition.
+     *
+     * @param swaggerContent OpenAPI Definition content
```

Comment:
Shall we add the @return as well to the java doc?

Reply:
Added in https://github.com/wso2/carbon-apimgt/pull/9927/commits/378729ec7b9401b4d317907185d27b13e98a52da